File: Client/src/app/content/notes/full-note/models/api/notes/note-structure-result.ts
import { UpdateNoteStructureWS } from 'src/app/core/models/signal-r/innerNote/update-note-structure-ws';
import { NoteUpdateIds } from './note-update-ids';

export interface NoteStructureResult {
  updates: UpdateNoteStructureWS;
  updateIds: NoteUpdateIds[];
}

File: Backend/NootsWEB/Common/DTO/Notes/FullNoteSyncContents/NoteStructureResult.cs
using Common.DTO.WebSockets.InnerNote;
using System.Collections.Generic;

namespace Common.DTO.Notes.FullNoteSyncContents;

public class NoteStructureResult
{
    public List<UpdateIds> UpdateIds { get; set; } = new List<UpdateIds>();

    public UpdateNoteStructureWS Updates { set; get; } = new();

    public NoteStructureResult()
    {

    }
}
